pages/event/create.tsx

```diff
@@ -1,7 +1,132 @@
-import React from "react";
+import React, { useState } from "react";
+import Image from "next/image";
+import { AccordionDefault } from "../../stories/Accordion/Accordion";
+import Select from "../../stories/Select/Select";
+import { timezones } from "../../mocks/timezones";
+import RadioButtons from "../../stories/Radio/Radio";
+import { DayPicker } from "../../stories/DayPicker/DayPicker";
+import { TimePicker } from "../../stories/TimePicker/TimePicker";
+import { InputText } from "../../stories/Input/InputText";
+import { SidebarDrawer } from "../../stories/SidebarDrawer/SidebarDrawer";
+import freepikCharacter from "../../public/img/freepik--Character--inject-25.svg";
+
+const options = [
+  { title: "Presencial", isDisabled: false },
+  { title: "Online", isDisabled: false },
+  { title: "Híbrido", isDisabled: false },
+];
+
+interface TimeOptions {
+  hour: "2-digit";
+  minute: "2-digit";
+}
 
 const CreateEvent = () => {
-  return <>CreateEvent</>;
+  const formatTimeOptions: TimeOptions = {
+    hour: "2-digit",
+    minute: "2-digit",
+  };
+  const DATE = new Date().toLocaleTimeString("es-ES", formatTimeOptions);
+  const [selected, setSelected] = useState<string>(options[0].title);
```

Comment:
Sin estado inicial, aparece el placeholder.. En componente `Select` podrias exponer el `onChange` y `value`, asi sacamos el `state` y se maneja por fuera
```suggestion
  const [selected, setSelected] = useState<string>();
```

Reply:
Bro no comprendo lo que me quieres decir aquí, cuando hablas de onChange y value te refires al componente Select o al componente Radio ?

Reply:
En este comment me referia al componente Select, si